content/developer/reference/user_interface/view_architectures/field_attribute_name.rst

```diff
@@ -5,3 +5,12 @@
 
    :requirement: Mandatory
    :type: str
+
+.. attribute:: widget
+   :noindex:
+
+   The widget used to represent the field. The selected widget can change the way the field is
+   rendered and/or the way it can be edited.
```

Comment:
not sure, but maybe mention that the widget is actually the key that will be looked up in the fields registry